src/mablApiClient.ts

```diff
@@ -6,10 +6,13 @@ import axios, {AxiosInstance, AxiosRequestConfig} from 'axios';
 import {Environment} from './entities/Environment';
 import {USER_AGENT} from './constants';
 
+const GET_REQUEST_TIMEOUT_MILLIS = 60_000;
+const POST_REQUEST_TIMEOUT_MILLIS = 900_000;
```

Comment:
@jbaldassari , is this the same timeout logic you added in the CLI, for deployments pulling?
I want to address the PR descr linked customer issue where this Action appears to hang.

Reply:
I think it's a little bit of a different situation in `mabl-cli` because we have the auto-retry logic in there too, but we're using very liberal timeouts now: https://github.com/mablhq/mabl-cli/blob/master/runtime/src/api/mablApiClient.ts#L127-L130

The individual request uses no timeout (`0`), and the retryable wrapper uses a 1 hour timeout